### PR TITLE
Allow link text in funding widget

### PR DESCRIPTION
**Pull request by sarahcrack**
### Trello card

https://trello.com/c/U5rbitfq/6926-allow-link-text-in-funding-widget

### Context

We want to be able to add link text to the subject-specific content in the funding widget on our [bursaries and scholarships](https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries) page.

### Changes proposed in this pull request

Adapted widget code to allow anchor tags by removing `html_safe` from `.erb` file and incorporating into `.rb` file for `funding_results`.

### Guidance to review
- check it's a suitable approach from a code perspective.
- check it has the intended behaviour needed.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -60,7 +60,7 @@ en:
         name: "Biology"
         group: "Secondary"
         sub_head: "Biology - Secondary"
-        funding: "Bursaries of %{bursaries_postgraduate_biology} are available for trainee biology teachers if you're eligible."
+        funding: "Bursaries of %{bursaries_postgraduate_biology} are available for <a href='/life-as-a-teacher/explore-subjects/biology'>trainee biology teachers</a> if you're eligible."
```
Comment (martyn-w) [inline config/locales/en.yml:63]: better to use double quotes for the attributes  - I think you can escape them like this:

```
funding: "Bursaries of %{bursaries_postgraduate_biology} are available for <a href=\"/life-as-a-teacher/explore-subjects/biology\">trainee biology teachers</a> if you're eligible."
```

**Review by martyn-w — APPROVED** [review]
👍  LGTM

**Comment by Sarah-DfE** [comment]
Looks good to me from a content point of view too. Presumably the link will be the same for each subject, just the name of the subject will change @sarahcrack ?

**Comment by sarahcrack** [comment]
> Looks good to me from a content point of view too. Presumably the link will be the same for each subject, just the name of the subject will change @sarahcrack ?

Hi @Sarah-DfE Yes you just need to go into the `config/locales/en.yml` file and add the anchor links under each subject where you would like to link to the relevant subject page :) You can use biology (funding section), line 63, as a template for this. Links are very similar and looks like you can just edit the subject name at the end of the url.

**Pull request merged**